source/atlas-cli-ephemeral-cluster.txt

```diff
@@ -0,0 +1,115 @@
+.. _atlas-cli-ephemeral-cluster:
+
+=====================================================
+Test Automations with Ephemeral Projects and Clusters
+=====================================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+This tutorial demonstrates how to quickly create and delete ephemeral 
+|service| projects and {+clusters+} using the {+atlas-cli+}. Ephemeral
+projects and {+clusters+} provide temporary testing environments that
+are isolated from your production projects and {+clusters+}. You can use
+ephemeral projects and {+clusters+} to test your automation scripts
+before using the scripts on production {+clusters+}.
+
+.. _atlas-cli-ephemeral-reqs:
+
+Prerequisites 
+-------------
+
+Before you begin, complete the following tasks: 
+
+- Create an |service| user account
+- :ref:`Create <atlas-organizations-create>` an 
+  :atlas:`Atlas organization </organizations-projects/>` and
+  note its organization ID. You can find the organization ID by running the :ref:`atlas-organizations-list` command.
+- :ref:`install-atlas-cli`.
+- :ref:`connect-atlas-cli`.
+
+Follow These Steps
+------------------
+
+.. procedure::
+   :style: normal
+
+   .. step:: Create the ephemeral project, {+cluster+}, and database user.
+
+      a. Run the :ref:`atlas-users-describe` command to find your 
+         |service| user ID. Replace <YOUR-EMAIL> with the email address
+         that is associated with your |service| user account.
+
+         .. code-block::
+
+            atlas users describe --username <YOUR-EMAIL>
+      
+      b. Run the :ref:`atlas-projects-create` command to create the
+         ephemeral project. Replace <ORG-ID> with the organization ID
+         and replace <YOUR-USER-ID> with your |service| user ID that
+         you retrieved in the previous step.
+
+         .. code-block::
+
+            atlas projects create myEphemeralProject --orgId <ORG-ID> --ownerId <YOUR-USER-ID>
+
+      c. Retrieve the project ID for the ephemeral project you created
+         from the response. In the following example response, the
+         project ID is 64933bde48add154124e343f.
```

Comment:
I think using some sort of formatting here will help users by calling this info out
```suggestion
         project ID is ``64933bde48add154124e343f``.
```